### ISSUE TITLE
attention_mask issue

### ISSUE THREAD

**Issue by TheEmber** [TabbyML/tabby#92]
**Describe the bug**
Got "attention mask and the pad token id were not set" when trying to use tabby with VSCode extension. When executing API requests or using playground everything working correct.
Console log when got error:
```
2023-04-12 11:44:42,190 DEBG 'server' stderr output:
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.

2023-04-12 11:44:45,657 DEBG 'server' stdout output:
INFO:     172.17.0.1:0 - "POST /v1/completions HTTP/1.1" 200 OK
```
Also got the same result when using CPU.

**Information about your GPU**
```
Wed Apr 12 14:29:45 2023
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 530.30.02              Driver Version: 531.18       CUDA Version: 12.1     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                  Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf            Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA GeForce RTX 3060 L...    On | 00000000:01:00.0 Off |                  N/A |
| N/A   46C    P8               12W /  N/A|    179MiB /  6144MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+

+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|    0   N/A  N/A        22      G   /Xwayland                                 N/A      |
+---------------------------------------------------------------------------------------+
```

**Additional context**
Run command:
```
sudo docker run \
  -it --rm \
  --gpus all \
  -v "/$(pwd)/data:/data" \
  -v "/$(pwd)/data/hf_cache:/home/app/.cache/huggingface" \
  -p 5000:5000 \
  -e MODEL_NAME=TabbyML/J-350M \
  --name=tabby \
  tabbyml/tabby
```
  
  Using WSL2, Ubuntu 22.04 distribution and Docker Desktop. GPU: Nvidia GeForce RTX 3060 Laptop.

**Comment by wsxiaoys** [comment]
Thank you for reporting the bug. Kindly ensure that you follow the instructions provided in the [README](https://github.com/TabbyML/tabby/blob/main/README.md) file and start tabby by executing the following command.

```bash
docker run \
  --gpus all \
  -it --rm \
  -v "/$(pwd)/data:/data" \
  -v "/$(pwd)/data/hf_cache:/home/app/.cache/huggingface" \
  -p 5000:5000 \
  -e MODEL_NAME=TabbyML/J-350M \
  -e MODEL_BACKEND=triton \
  --name=tabby \
  tabbyml/tabby
```

Removing MODEL_BACKEND=triton is not a supported method of starting tabby at the moment.

**Comment by TheEmber** [comment]
attention_mask issue actually dont blocking execution of program. So actual problem for me is VSCode for some reason does not show hint, but recieves responce from tabby.
VSCode devtools console output:
```
[Extension Host] Requesting:  {
  uuid: 1681470248210,
  timestamp: 1681470588032,
  prompt: 'def fibonacci(n):',
  language: 'python'
}
workbench.desktop.main.js:90 [Extension Host] Result completions:  [
  It {
    insertText: '\n' +
      '    if n == 0:\n' +
      '        return 0\n' +
      '    elif n == 1:\n' +
      '        return 1\n' +
      '    else:\n' +
      '        return fibonacci(n-1) + fibonacci(n-2)\n',
    range: r { c: [o], e: [o] },
    command: {
      title: 'Tabby: Emit Event',
      command: 'tabby.emitEvent',
      arguments: [Array]
    }
  }
]

```

**Comment by wsxiaoys** [comment]
> attention_mask issue actually dont blocking execution of program. So actual problem for me is VSCode for some reason does not show hint, but recieves responce from tabby. VSCode devtools console output:
> 
> ```
> [Extension Host] Requesting:  {
>   uuid: 1681470248210,
>   timestamp: 1681470588032,
>   prompt: 'def fibonacci(n):',
>   language: 'python'
> }
> workbench.desktop.main.js:90 [Extension Host] Result completions:  [
>   It {
>     insertText: '\n' +
>       '    if n == 0:\n' +
>       '        return 0\n' +
>       '    elif n == 1:\n' +
>       '        return 1\n' +
>       '    else:\n' +
>       '        return fibonacci(n-1) + fibonacci(n-2)\n',
>     range: r { c: [o], e: [o] },
>     command: {
>       title: 'Tabby: Emit Event',
>       command: 'tabby.emitEvent',
>       arguments: [Array]
>     }
>   }
> ]
> ```

are you using the triton backend, as suggested in above issue?

**Comment by TheEmber** [comment]
Since tabby is works for me now issue can be closed, but I dont sure why it works only in Insiders version of VSCode.
